src/textual/widgets/_tabbed_content.py

```diff
@@ -280,7 +260,7 @@ def add_pane(
         *,
         before: TabPane | str | None = None,
         after: TabPane | str | None = None,
-    ) -> AwaitTabbedContent:
+    ) -> AwaitComplete:
```

Comment:
Do we know the return type here?

Reply:
Actually, the plan to use generics with this is a bit flawed as it stands because the `AwaitComplete` can take multiple coroutines, and they may have different return types. We may not want to pass the return values from all of the coroutines up.

I think to add support for generics would require extra API work for specifying the return value. For now it might be better to just not support the generic since it's never used, and if we have need for it in the future, add it?

Reply:
I think that's reasonable. If all we are doing it awaiting until it is ready, then no return type needed.